proposals/0013-wave-size-range.md

```diff
@@ -83,11 +83,18 @@ Where `minWaveSize` is the minimum wave size supported by the shader
 
 ### DXIL Additions
 
-The existing `WaveSize` value is stored as a metadata 32-bit integer constant.
-To store the additional values, an additional metadata tag indicating a tuple of
- three 32-bit integers representing the given parameters to `WaveSize`.
-In the case where no preferred wave size is specified, the value zero will
- indicate that nothing was specified.
+The existing `WaveSize` value is stored as a single metadata 32-bit integer
+ constant.
+The new metadata tag expands this to a tuple of three 32-bit integers.
+These values represent and and all parameters to `WaveSize`.
```

Comment:
and typo

Reply:
woops! Not entirely sure what I was going to put here, but I'll put something useful in its place.